peachjam/templates/peachjam/layouts/main.html

```diff
@@ -24,13 +24,17 @@
     {% if PEACHJAM_SETTINGS.google_analytics_id %}
       <!-- Global site tag (gtag.js) - Google Analytics -->
       <script async
-              src="https://www.googletagmanager.com/gtag/js?id={{ PEACHJAM_SETTINGS.google_analytics_id }}"></script>
+              src="https://www.googletagmanager.com/gtag/js?id={{ PEACHJAM_SETTINGS.google_analytics_id|split|first }}"></script>
```

Comment:
@longhotsummer We also have this tag that loads the tag manager script that uses the analytics ID. I have used the first google tag id for this. All examples I can see just use one id to load the script.

Reply:
ja, i didn't know what to do here either.